packages/colors/src/tsconfig.json

```diff
@@ -1,9 +1,6 @@
 {
     "extends": "../../../config/tsconfig.base",
     "compilerOptions": {
-        "lib": ["es6", "dom"],
-        "module": "commonjs",
-        "outDir": "../lib",
-        "target": "ES2015"
```

Comment:
I believe [it’s](https://github.com/palantir/blueprint/pull/6179#issuecomment-1563264322) is caused by this. Instead of targeting ES2015, it now targets ES5.

Reply:
I'll add in the `tslib` dependency. It's better for the packages to be consistent. We can consider migrating to `"target": "es2015"` in a future major version.